estuary/api/v1.py

```diff
@@ -218,3 +222,42 @@ def _get_partial_stories(query, reverse=False):
         all_results.append(rv)
 
     return jsonify(all_results)
+
+
+@api_v1.route('/siblings/<resource>/<uid>')
+def get_siblings(resource, uid):
+    """
+    Get siblings of next/previous node that are correlated to the node in question.
+
+    :param str resource: a resource name that maps to a neomodel class
+    :param str uid: the value of the UniqueIdProperty to query with
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    :raises NotFound: if the item is not found
+    :raises ValidationError: if an invalid resource was requested
+    """
+    # If last is True, we fetch siblings of the next node, previous node otherwise
+    last = request.args.get('last', False)
+
+    item = get_neo4j_node(resource, uid)
+    if not item:
+        raise NotFound('This item does not exist')
+
+    item_story_flow = story_flow(item.__label__)
+    if not item_story_flow['backward_label'] or not item_story_flow['forward_label']:
+        raise ValidationError('Siblings cannot be determined on this kind of resource')
+
+    if last:
+        correlated_nodes = get_correlated_node(item_story_flow['forward_label'], item, last=True)
```

Comment:
In this context `get_correlated_node` doesn't sound like the proper name. Perhaps make this `get_correlated_nodes` and rename `get_correlated_nodes` to `get_all_correlated_nodes_count`.